Sources/STTextViewUIKit/STTextView.swift

```diff
@@ -436,7 +436,7 @@ import STTextViewCommon
     }
 
     open override var canBecomeFirstResponder: Bool {
-        !isFirstResponder && isEditable
+        !isFirstResponder
```

Comment:
```suggestion
        !isFirstResponder && isSelectable
```

do you think it should check for `isSelectable`? That condition would match the Mac version